src/irsForms/Form.ts

```diff
@@ -6,7 +6,9 @@ export type FormTag =
   'f1040sb' |
   'f1040sd' |
   'f1040se' |
-  'f1040sei'
+  'f1040sei' |
+  'f1040s8'
```

Comment:
I was about to mention this should be the full number, but you've accurately captured the filename as it's downloaded from IRS, so this is good!

Reply:
Yep, I just copied the IRS filename.